src/Build.UnitTests/Evaluation/ProjectSdkImplicitImport_Tests.cs

```diff
@@ -554,7 +554,7 @@ private void VerifyPropertyFromImplicitImport(Project project, string propertyNa
 
         private SdkReference GetParsedSdk(ProjectImportElement element)
         {
-            PropertyInfo parsedSdkInfo = typeof(ProjectImportElement).GetProperty("ParsedSdkReference", BindingFlags.Instance | BindingFlags.NonPublic);
+            PropertyInfo parsedSdkInfo = typeof(ProjectImportElement).GetProperty("SdkReference", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
I like the rename, but it makes the more interesting change in this PR hard to review. Can you separate it into its own commit?

Reply:
Done!